doc/api/process.md

```diff
@@ -1911,18 +1911,19 @@ Will generate an object similar to:
 
 <!-- eslint-skip -->
 ```js
-{
-  http_parser: '2.3.0',
-  node: '1.1.1',
-  v8: '6.1.534.42-node.0',
-  uv: '1.3.0',
-  zlib: '1.2.8',
-  ares: '1.10.0-DEV',
-  modules: '43',
-  icu: '55.1',
-  openssl: '1.0.1k',
-  unicode: '8.0',
-  cldr: '29.0',
+{ http_parser: '2.7.0',
+  node: '10.0.0-pre',
```

Comment:
Really minor nit, but I suggest using an actual release version to avoid any confusion from the `-pre`.